modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/ding_ddbasic_opening_hours.module

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @file
+ * Extending opening hours to match the ddbasic theme.
```

Comment:
The field added is an extra_field, so I can't use the field formatter like we talked about. But we're no longer removing any files from the original opening hours module, so one could change between the ddbasic weekly pane and standard weekly pane. I've tested they both work.